project/src/main/java/com/google/sps/data/ProductSearchLibrary.java

```diff
@@ -211,15 +182,9 @@ public static void removeProductFromProductSet(
             ProductSearchClient.formatProductSetName(projectId, computeRegion, productSetId);
         // List all the products available in the product set.
         for (Product product : client.listProductsInProductSet(formattedName).iterateAll()) {
-            ProductItem productItem = new ProductItem(product.getName().substring(product.getName().lastIndexOf('/') + 1), product.getDisplayName(),
-                                        product.getProductCategory());
-            productItems.add(productItem);
-            for (Product.KeyValue element : product.getProductLabelsList()) {
-                System.out.println(String.format("%s: %s", element.getKey(), element.getValue()));
-            }
+            createProductItemAndAddToList(product.getName().substring(product.getName().lastIndexOf('/') + 1), product.getDisplayName(),
```

Comment:
I would add a comment here about what `product.getName().substring(product.getName().lastIndexOf('/') + 1)` is for. You can just say that it's being used as the product ID.